src/components/WordCountProgressBar.tsx

```diff
@@ -11,7 +11,7 @@ export default function WordCountProgressBar({ current, target, period }: Props)
     <div>
       <Progress value={(current / target) * 100} variant="filled" />
       <p>
-        {current} / {target} words {period}
+        {current} / {target} words {period.replace(/["]+/g, "")}
```

Comment:
Why is the `replace` necessary?

Reply:
The copy was showing up as "100/500 words "daily""